projects/services-ogc/src/lib/opensearch/README.md

```diff
@@ -0,0 +1,6 @@
+## Native support
+
+Most browser have a method `window.external.AddSearchProvider` that speaks opensearch.
```

Comment:
Window.external is **Deprecated** - https://developer.mozilla.org/en-US/docs/Web/API/Window/external

Reply:
Thanks for the link! Removed.